test/testdoT.js

```diff
@@ -61,4 +61,15 @@ describe('doT', function(){
 		});
 	});
 
+	describe('whitespace stripping', function() {
+		function assert_match(actual, expected) {
+			assert(actual.match(expected), JSON.stringify(actual) + " did not match " + expected);
+		}
+		it('should replace stretches of whitespace with one whitespace', function() {
+			assert_match(doT.template('<div>    </div>')(), /^<div>\s<\/div>$/);
```

Comment:
Why didn't you use `assert.equal`? You know the value exactly. No need to regex match it.